### DISCUSSION TITLE
Where did the icons for material-forum-editor-buttons come from?

### DISCUSSION THREAD

**Discussion by CST1229** [ScratchAddons/ScratchAddons#3135] [Help] [answered]
I'm just asking since I'm making an addon that adds a forum editor button and I'd like to make it compatible with that and be consistent.

**Comment by mxmou** [comment, marked as answer]
Probably https://fonts.google.com/icons

**Reply by CST1229** [reply]
Looks like so, found matches for everything except scratchblocks (probably custom-made) and list items (easily creatable from the unordered list icon).
